VEX/priv/host_riscv64_isel.c

```diff
@@ -1551,6 +1551,19 @@ static HReg iselFltExpr_wrk(ISelEnv* env, IRExpr* e)
       return dst;
    }
 
+   /* ---------------------- MULTIPLEX ---------------------- */
+   case Iex_ITE: {
+      /* ITE(ccexpr, iftrue, iffalse) */
+      HReg   cond    = iselIntExpr_R(env, e->Iex.ITE.cond);
+      HReg   iftrue  = iselFltExpr(env, e->Iex.ITE.iftrue);
+      HReg   iffalse = iselFltExpr(env, e->Iex.ITE.iffalse);
+      HReg   dst     = newVRegF(env);
+      IRType csel_ty = typeOfIRExpr(env->type_env, e->Iex.ITE.iftrue);
+      vassert(csel_ty == Ity_F64 || csel_ty == Ity_F32);
```

Comment:
Nit: If an IR expression cannot be lowered then the control should go to the irreducible label. It additionally prints information about what IR could not be processed.

Reply:
Hmm, function `iselFltExpr_wrk()` already previously checks that the type is `Ity_F64` or `Ity_F32`. It might be then best to remove this assertion as `csel_ty` should not be needed either.

Reply:
Ok, we will refine this PR :)